File: 251004/Yelkin/TopicService/src/main/java/by/yelkin/TopicService/entity/Creator.java
package by.yelkin.TopicService.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "tbl_creator")
public class Creator extends BaseEntity {
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 251004/Marchenko/lab1/DistComp_1/Models/Creator.cs
namespace DistComp_1.Models;

public class Creator : BaseModel
{
    public string Login { get; set; }
    public string Password { get; set; }
    public string Firstname { get; set; }
    public string Lastname { get; set; }

    public List<Issue> Issues { get; set; } = [];
}